tests/cpp/gbm/test_gbtree.cc

```diff
@@ -294,6 +305,14 @@ TEST(Dart, Prediction) {
   }
 }
 
+TEST(Dart, Prediction) {
+  TestDartPrediction("auto");
+  TestDartPrediction("cpu_predictor");
+#if defined(XGBOOST_USE_CUDA)
+  TestDartPrediction("gpu_predictor");
+#endif  // defined(XGBOOST_USE_CUDA)
+}
+
```

Comment:
Nitpick: It might be better to use [parameterized tests](https://github.com/google/googletest/blob/main/docs/advanced.md#value-parameterized-tests), so that we can identify which of the three tests fail. Currently, you'd see a single `FAIL` output when one of `TestDartPrediction` calls fail.

Reply:
Changed to parameterized tests.